README.md

```diff
@@ -7,23 +7,27 @@ Laravel package to provide fair consumption of jobs against multiple partitions.
 composer require aloware/fair-queue
 ```
 
+## Assets
+Run the command below to publish the package asset files:
+
+```sh
+php artisan vendor:publish --tag=public --force
+```
+
+Run the command below to publish the package config file:
+
+```sh
+php artisan vendor:publish --tag=fairqueue-config --force
+```
+
+
 ## Usage
 This package uses Redis as storage. So you need to add **fair-queue** database configuration
```

Comment:
This package uses Redis as data storage. By default it uses "default" redis connection. You can configure it to the connection of your choice within the fair-queue configuration file or by setting in the environment file.